scripts/js_texts/extracted_strings_app.js

```diff
@@ -122,16 +140,60 @@ module.exports = [
     'High',
     'High Barrier',
     'High Tick',
+    'High Tick/Low Tick contracts have a strict duration of five ticks.',
     'High barrier',
+    'High barrier must be higher than low barrier',
+    'High barrier offset',
+    'High, Low and Close',
     'High-Close',
     'High-Low',
     'Higher',
+    'Higher/Lower contracts will be refunded at the purchase price if there are less than 2 ticks between the start and end times.',
     'Hour',
     'I want to reapply',
     'IDK',
+    'If the <strong>exit spot</strong> is equal to either the <strong>Low barrier</strong> or the <strong>High barrier</strong>, you don\'t win the payout.',
+    'If the <strong>exit spot</strong> is equal to the <strong>barrier</strong> or the <stronm g>new barrier (if a reset occurs)</stronmg>, you don\'t win the payout.',
```

Comment:
`<stronm g>new barrier (if a reset occurs)</stronmg>`
typo

Reply:
Seems like the typo is fixed in the component. Probably, the strings should be generated again to fix it.